site.mk

```diff
@@ -15,19 +14,11 @@ GLUON_FEATURES := \
 	web-private-wifi \
 	web-logging \
 
-
 GLUON_SITE_PACKAGES := \
 	iwinfo \
-	iptables \
 	haveged \
-	ffffm-banner_legacy \
-	ffffm-keep-radio-channel \
-	ffffm-button-bind \
```

Comment:
While i think removing button-bind is a good idea, as the behavior is not consistent across platforms (hardware driven switch-leds, uncontrollable Power-LEDs), a cleanup of selected config states should be done (if this package has a persistent state).

Reply:
The pure "ffffm-button-bind" is a very helpfull package to protect node maintainer to search a proplem that doesn't really exist ([Freifunk housekeeper effect ](https://forum.freifunk.net/t/firmware-anderungswunsch-wifi-taste-deaktivieren/9009)).
"ffffm-button-bind" is one of the first simple base version of that kind of packages. Please do not compare it with the extended Gluon PR version.

Reply:
"ffffm-banner" contains two tool scripts (nodeinfo / meshinfo) for the analysis of node and mesh topologies they should not be underestimated.

Reply:
Regarding the tool scripts and button bind: I don't really see why we need our own tools for this, would prefer going into the migration with as little (custom) overhead as possible, and Upstream the tools/button bind if helpful (personally never used them).

Reply:
Regarding the **tool scripts**: There is no comparable script upstream now. The scripts are a collection of useful snippets from other communities and have been expanded based on own experience and needs as mesh maintainers.

Regarding **button-bind**: There is an upstream PR. This PR based on my/our Package. Currently the PR is more unwieldy and not very user friendly in my opinion.